src/main/xerus/monstercat/api/response/Track.kt

```diff
@@ -35,10 +28,7 @@ open class Track: MusicItem() {
 	var albumArtists = ""
 	var albumId = ""
 	var albumName = ""
-	var trackNumber = -1
-	
-	val streamHash: String?
-		get() = albums.find { it.streamHash.isNotEmpty() }?.streamHash
+	@Key var trackNumber = -1
```

Comment:
how come that this is a Key now? Then it should be moved up to the other keys

Reply:
165671b5eb19d28ade7c1914e43b15ead407e3bd

Reply:
25c4d09eabf4bdc897e8bfb713ffb8cede0b930c